diff --git a/init.d b/init.d
index b2eec60..50650c9 100644
--- a/init.d
+++ b/init.d
@@ -57,7 +57,7 @@ case "$1" in
 	modprobe capability >/dev/null 2>&1 || true
 	if [ ! -f /etc/bind/rndc.key ]; then
 	    /usr/sbin/rndc-confgen -a -b 512 -r /dev/urandom
-	    chmod 0640 /etc/bind/rndc.key
+	    chmod 0644 /etc/bind/rndc.key
 	fi
 	if [ -f /var/run/named/named.pid ]; then
 	    ps `cat /var/run/named/named.pid` > /dev/null && exit 1
